akka-sample-sharding-scala/src/main/scala/sample/sharding/TemperatureService.scala

```diff
@@ -0,0 +1,64 @@
+package sample.sharding
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+import akka.actor.typed.{ ActorRef, Behavior }
+import akka.actor.typed.scaladsl.Behaviors
+import akka.cluster.sharding.typed.ShardingEnvelope
+import akka.cluster.sharding.typed.scaladsl.{ ClusterSharding, Entity, EntityTypeKey }
+
+object TemperatureService {
+
+  trait TemperatureEvent
+  // Update a random device
+  case object UpdateDevice extends TemperatureEvent
+  case object ReadTemperatures extends TemperatureEvent
+
+  val TypeKey: EntityTypeKey[Device.Command] =
+    EntityTypeKey[Device.Command]("Device")
+
+  def apply(): Behavior[TemperatureEvent] =
+    Behaviors.setup { context =>
+
+      val random = new Random()
+      val numberOfDevices = 50
```

Comment:
constant so can move out in the object and start with upper case letter

Reply:
In the previous version it was hard coded like this in the actor. I just transferred it as is. But it's better in the object, moved.